Images/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Athaliah Portflio SetUp</title>
+ <link rel="stylesheet" href="style.css">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <!--ToolBar-->
+   
+    <header>
+ <div class="container " >
+    
+    <div class="wrapper"> 
+        <nav id="nav">
+            
+            <div class="myLogo">      
+<image src="./logo.svg" width="Hug (76px)" >
+    </image><div>
+</nav>
+</header>
+<div class="container" onclick="myFunction(this)">
+    <div class="bar1"></div>
+    <div class="bar2"></div>
+    <div class="bar3"></div>
+
+<!--Main-Content-->
+  </div>
+</div>
+ </div>
+ <section id="Main_Content">
+ <div class="header-container">
+    <div class="wrapper">
+
+<div class="hero-content">
+    
+<h1>I'm Athaliah Glad to see you!</h1>  
+<p>
+I'm a software developer ! I can help you build a product , 
+<br>feature or website Look through some of my work and experience!<br>
+If you like what you see and have a project you need coded, don't hestiate to contact me.
+
+</p>
+<footer>
+    
+<h2>LET'S CONNECT</h2>
```

Comment:
Please fix this to be a link `<a>`. By its a color, you can see that its blue. This should redirect people visiting your page to a contact form or your socials.